ceph_deploy/util/constants.py

```diff
@@ -13,3 +13,12 @@
 mds_path = join(base_path, 'mds')
 
 osd_path = join(base_path, 'osd')
+
+# Default package components to install
+default_components = (
+    'ceph-osd',
+    'ceph-radosgw',
```

Comment:
As you noticed in another thread, this package name is actually different on DEB vs RPM systems.  RPM == ceph-radosgw, and DEB == radosgw.

Even though the package split is not in Debian yet, do we want to handle that now?  Should there be an rpm_default_components and deb_default_components?

Or should there maybe be a get_default_components() method that takes in a host type param, and can then return a different tuple based on the type of host?